GPflowOpt/design.py

```diff
@@ -26,6 +28,11 @@ def __init__(self, size, domain):
         self.domain = domain
 
     def generate(self):
+        """
+        Returns a design of the requested size (N) and domain dimensionality (D). All data points are contained
```

Comment:
remove requested. seems to imply N and D are parameters of generate (instead of the CTor?)